mod/Jailbreak.Logs/Listeners/LogEntityListeners.cs

```diff
@@ -15,6 +16,24 @@ public class LogEntityListeners : IPluginBehavior
 	public LogEntityListeners(IRichLogService logs)
 	{
 		_logs = logs;
+		OnDropGun();
+	}
+	
+	private static MemoryFunctionVoid<CBasePlayerPawn, CBasePlayerWeapon, IntPtr> WeaponDrop = new( "\\x55\\x48\\x89\\xE5\\x41\\x56\\x41\\x55\\x49\\x89\\xD5\\x41\\x54\\x49\\x89\\xFC\\x53\\x48\\x89\\xF3\\xE8\\x2A\\x2A\\x2A\\x2A" );
+
+	private void OnDropGun()
+	{
+		WeaponDrop.Hook(hook =>
+		{
+			var player = hook.GetParam<CBasePlayerPawn>(0);
+			var weapon = hook.GetParam<CBasePlayerWeapon>(1);
+
+			if (!player.IsValid) return HookResult.Continue;
+			if (player.TeamNum != (byte)CsTeam.CounterTerrorist) return HookResult.Changed;
```

Comment:
Please use player.Team instead of player.TeamNum

Reply:
@MSWS Should be fixed. Needs testing for the weapon pickup